packages/rest-explorer/src/__tests__/acceptance/rest-explorer.acceptance.ts

```diff
@@ -107,7 +107,7 @@ describe('API Explorer (acceptance)', () => {
       config: RestExplorerConfig,
     ) {
       app = givenRestApplication();
-      app.bind(RestExplorerBindings.CONFIG).to(config);
+      app.configure(RestExplorerBindings.COMPONENT).to(config);
```

Comment:
Please preserve this test to verify that `app.bind(RestExplorerBindings.CONFIG).to(config);` keeps working with your changes in place and add a new test to verify `app.configure` flavor.

Do we care about users that may be using the key `rest-explorer.config` directly?

```ts
app.bind('rest-explorer.config').to(config);
```

The code above works with the current Explorer version but will break with your changes in place.